### ISSUE TITLE
By clicking on home option of the footer the page says that page is not found...I want to solve this issue.

### ISSUE THREAD

**Issue by AnitSarkar123** [AdityaSeth777/PymPay#11]
Assign me under Aperture'24

**Comment by AdityaSeth777** [comment]
Alright. Assigning. Make sure to change the other footer links too, because adityaseth.me is now changed to adityaseth.in
The sub links remain the same.

**Comment by AdityaSeth777** [comment]
Assigned. Happy Contributing !

**Comment by AnitSarkar123** [comment]
Is this issue under Aperture'24... because tag is not present..and is this issue easy, medium or hard

**Comment by AdityaSeth777** [comment]
Yes it is. This issue is Easy. All the details are mentioned in the server. Please check.
